e2e-tests/tfm/cypress/e2e/journeys/login/login.spec.js

```diff
@@ -82,23 +113,25 @@ context('User can login', () => {
     });
   });
 
-  it('should not login and redirect to /deals with invalid email/username', () => {
-    pages.landingPage.email().type('wrongUser');
-    pages.landingPage.submitButton().click();
-    cy.url().should('eq', relative('/'));
+  it('landing page without user redirects to https://login.microsoftonline.com', () => {
+    cy.request({
+      url: relative('/'),
+      followRedirect: false, // avoid infinite redirect loop
+    }).then((resp) => {
+      expect(resp.status).to.eq(302);
+      expect(resp.redirectedToUrl).match(/^https:\/\/login\.microsoftonline\.com\/*/);
+    });
   });
 
-  it('should show relevant header items when logged out', () => {
-    partials.header.ukefLogo().should('exist');
-    partials.header.headerName().should('exist');
-    partials.header.headerName().contains('Trade Finance Manager');
-
-    partials.header.userLink().should('not.exist');
-    partials.header.signOutLink().should('not.exist');
+  it('logout redirects to https://login.microsoftonline.com', () => {
+    cy.mockLogin(T1_USER_1);
 
-    partials.header.betaBanner().should('exist');
-
-    partials.primaryNavigation.allDealsLink().should('not.exist');
-    partials.primaryNavigation.allFacilitiesLink().should('not.exist');
+    cy.request({
+      url: relative('/logout'),
+      followRedirect: false, // avoid infinite redirect loop
+    }).then((resp) => {
+      expect(resp.status).to.eq(302);
+      expect(resp.redirectedToUrl).match(/^https:\/\/login\.microsoftonline\.com\/*/);
+    });
```

Comment:
I think we can split up these tests a little bit more for readability and if something fails, it's easier to debug e.g

```js
beforeEach(() => {
  cy.mockLogin(T1_USER_1);
});

it('should redirect to X', () => ...

it("should display the user's name in the header", () => ...
```

Rather than having multiple assertions in a single "it()"

Also, should `https://login.microsoftonline.com` be a constant or env var?

Reply:
Splitting sounds good and we need new constants.